deeppavlov/models/morpho_syntax_parser/syntax_parsing.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2019 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Optional, Tuple, Union
+
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.component import Component
+
+
+def make_pos_and_tag(tag: str, sep: str = ",",
+                     return_mode: Optional[str] = None) -> Tuple[str, Union[str, list, dict, tuple]]:
+    """
+    Args:
+        tag: the part-of-speech tag
+        sep: the separator between part-of-speech tag and grammatical features
+        return_mode: the type of return value, can be None, list, dict or sorted_items
+    Returns:
+        the part-of-speech label and grammatical features in required format
+    """
+    if tag.endswith(" _"):
+        tag = tag[:-2]
+    if sep in tag:
+        pos, tag = tag.split(sep, maxsplit=1)
+    else:
+        pos, tag = tag, ("_" if return_mode is None else "")
+    if return_mode in ["dict", "list", "sorted_items"]:
+        tag = tag.split("|") if tag != "" else []
+        if return_mode in ["dict", "sorted_items"]:
+            tag = dict(tuple(elem.split("=")) for elem in tag)
+            if return_mode == "sorted_items":
+                tag = tuple(sorted(tag.items()))
+    return pos, tag
+
+
+@register('dependency_output_prettifier')
+class DependencyOutputPrettifier(Component):
+    """Class which prettifies dependency parser output
+    to 10-column (Universal Dependencies) format.
+    Args:
+        return_string: whether to return a list of strings or a single string
+        begin: a string to append in the beginning
+        end: a string to append in the end
+        sep: separator between word analyses
+    """
+
+    def __init__(self, return_string: bool = True, begin: str = "",
```

Comment:
should return_string be parameter? In which case we will set it False?

Reply:
return_string is False in check_models function of models.morpho_tagger_parser.joint:JointTaggerParser component